daemon/algod/api/algod.oas2.json

```diff
@@ -3639,6 +3643,22 @@
           "x-algorand-format": "uint64"
         }
       }
+    },
+    "SimulationEvalChanges": {
+      "description": "The set of parameters and limits override during simulation.",
```

Comment:
Could you also mention that if this object is present, evaluation paramters differed in some way from standard evaluation?

Also, this will be the last name change suggestion I promise 😄: what about `SimulationEvalOverrides` for this? I think it would be consistent with `simulation.ResultEvalOverrides`, and "changes" could possibly be misinterpreted as state changes

Reply:
see [updated eval test, repeat 40 times line 1050 length, eval-changes to …](https://github.com/algorand/go-algorand/pull/5247/commits/04436a3ebb3923b05ccc7a0614f7d4bc7254e057), changed to `SimulationEvalOverrides` and `eval-changes` to `eval-overrides`.